### PR TITLE
add two values to modify via settings

### PR DESCRIPTION
**Pull request by Calibretto22**
My changes enable two additional values of the roboport to be edited in the mod menu: 

- data.raw.roboport.roboport.energy_usage (idle energy consumption)
- data.raw.roboport.roboport.energy_source.buffer_capacity (internal battery)

Initially both settings are left with their default values. The settings can be used to either buff or nerf the roboports.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ All variables can be changed via the ingame mod settings (needs game relaunch).
 
 ## Default Changes to Base Game
 
-* Charging Speed per Robot `1MW` -> `10MW`
+* Charging Speed per Robot `1MW` -> `4MW`
```
Comment (Anthrazz) [inline Readme.md]: You change the value in the settings.lua to `1MW`.  Im fine with this as it is the game default. I would just note this here with something like this:
```suggestion
* Charging Speed per Robot stays at default `1MW`
```

**Review by Anthrazz — CHANGES_REQUESTED** [review]
Thank you for the addition. This mostly looks good for me. I have added just a small suggestion.

File: Readme.md (hunk 1 above)
Comment (Calibretto22) [reply Readme.md]: Yes. I made this change and did not reflect it in the documentation. Your correction is totally fine.

**Comment by Calibretto22** [comment]
Damn. I'm a dummy with github and its workflows. 
I guess i have to make the change in my Repo to include it in the pull request. This was not obvious to me.

**Review by Anthrazz — APPROVED** [review]

**Comment by Anthrazz** [comment]
Thank you again for the changes. I have merged it and uploaded the new version to the mod page.

**Pull request merged**